docs/supported-libraries.md

```diff
@@ -138,8 +138,8 @@ These are the JVMs and operating systems that the integration tests are run agai
 
 | JVM                                               | Versions  | OS                             |
 | ------------------------------------------------- | --------- | ------------------------------ |
-| [AdoptOpenJDK Hotspot](https://adoptopenjdk.net/) | 8, 11, 15 | Ubuntu 18, Windows Server 2019 |
-| [AdoptOpenJDK OpenJ9](https://adoptopenjdk.net/)  | 8, 11, 15 | Ubuntu 18, Windows Server 2019 |
+| [AdoptOpenJDK Hotspot](https://adoptopenjdk.net/) | 8, 11, 17 | Ubuntu 18, Windows Server 2019 |
```

Comment:
as we are not really using builds from AdoptOpenJDK maybe change these to
`[Eclipse Temurin, Hotspot](https://adoptium.net/)` Maybe also use OpenJDK instead of Hotspot?
`[IBM Semeru Runtimes, OpenJ9](https://developer.ibm.com/languages/java/semeru-runtimes/)`